nucypher/cli/commands/status.py

```diff
@@ -179,7 +178,7 @@ def events(general_config, registry_options, contract_name, from_block, to_block
         if event_name:
             raise click.BadOptionUsage(option_name='--event-name', message='--event-name requires --contract-name')
         # FIXME should we force a contract name to be specified?
-        contract_names = [STAKING_ESCROW_CONTRACT_NAME, POLICY_MANAGER_CONTRACT_NAME]
+        contract_names = [STAKING_ESCROW_CONTRACT_NAME,]
```

Comment:
Looking into fixing this - see https://github.com/KPrasch/nucypher/pull/35 and https://github.com/nucypher/nucypher/pull/2867

Reply:
oh 😮

Reply:
Feel free to review https://github.com/KPrasch/nucypher/pull/35. I couldn't add other reviewers because it is a PR into @KPrasch 's branch, but extra eyes would be helpful 🙏 .